tests/js/function-caller.test.js

```diff
@@ -105,14 +94,20 @@ describe('Test functionCaller', () => {
 
 
   test('will not return AWS S3 Key if return value from callable not supplied', async () => {
+    // TODO: check S3 response object
+
+    let testKey = 'test/key';
 
-    AWS.S3.upload = jest.fn();
```

Comment:
I removed this because it didn't work as originally intended. Left a TODO and used aws-sdk-mock instead.